frontend/src/Components/CatalogInput.js

```diff
@@ -0,0 +1,253 @@
+import React from 'react';
+import { Button, Card, Row, Col, Modal, Form } from 'react-bootstrap';
+import { Container as FABContainer, Button as FAButton } from 'react-floating-action-button'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faPlus, faTimes } from '@fortawesome/free-solid-svg-icons'
+
+class CatalogInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+        showAddForm: false,
+        addForm: {
+          'serviceName': '',
+          'serviceDescription': '',
+          'serviceImageURL': '',
+        },
+        showEditForm: false,
+        editForm: {
+            'serviceName': '',
+            'serviceDescription': '',
+            'serviceImageURL': '',  
+        },
```

Comment:
createForm, updateForm might be better

Reply:
The component asks the user to add a product, instead of creating a product (which might sound off in this context). As these objects refer to that functionality, I believe it's fine to distance this part from the REST naming convention, especially since the data in these objects is processed multiple times before reaching the API endpoint.

Reply:
Oh Alright. This makes sense to keep it this way then 🤔